face_recognition_LDA.m

```diff
@@ -0,0 +1,32 @@
+
+% get directories for all images
+% please use 'getfn.m' with this script
+fn = getfn('C:\Users\json13\Desktop\att_faces', 'pgm$')
+
+% read and convert images to column vector
```

Comment:
Does this work on multiple folders? I have a data structure in the original file

Reply:
yes, it works on multiple folders. getfn function gives you directories for all images.